tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -429,211 +429,180 @@ def test_k_rc_pod_fail(self):
+                'output': del_output, 'bkts_to_del': del_bucket}
+        thread_del = threading.Thread(target=self.ha_obj.put_get_delete,
+                                      args=(event, s3_test_obj,), kwargs=args)
+        thread_del.daemon = True  # Daemonize thread
+        thread_del.start()
+        LOGGER.info("Step 3: Successfully started DELETEs in background for %s buckets",
+                    del_bucket)
 
-    # pylint: disable=too-many-statements
-    # pylint: disable=too-many-locals
-    @pytest.mark.ha
-    @pytest.mark.lc
-    @pytest.mark.tags("TEST-35791")
-    def test_cont_ios_during_server_kpods_down(self):
-        """
-        Test to verify continuous IOs while k server pods are failing one by one by delete
-        deployment
-        """
-        LOGGER.info("STARTED: Test to verify continuous IOs while k server pods are failing one "
-                    "by one")
+        LOGGER.info("Step 4: Perform WRITEs with variable object sizes in background")
+        test_prefix_write = 'test-write-35790'
+        output_wr = Queue()
+        event_set_clr = [False]
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': test_prefix_write,
+                'nclients': 1, 'nsamples': 7, 'skipread': True, 'skipcleanup': True,
+                'output': output_wr, 'event_set_clr': event_set_clr}
+        thread_wri = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,),
+                                      kwargs=args)
+        thread_wri.daemon = True  # Daemonize thread
+        thread_wri.start()
+        LOGGER.info("Step 4: Successfully started WRITEs with variable sizes objects"
+                    " in background")
 
-        LOGGER.info("Step 1: Perform Continuous WRITEs-READs-verify during k server pods are "
-                    "going down")
-        users = self.mgnt_ops.create_account_users(nusers=1)
-        self.test_prefix = 'test-35791'
-        self.s3_clean = users
-        event = threading.Event()
-        output = Queue()
-        args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
-                'nclients': 1, 'nsamples': 5, 'skipcleanup': False, 'output': output}
-        thread = threading.Thread(target=self.ha_obj.event_s3_operation,
-                                  args=(event,), kwargs=args)
-        thread.daemon = True  # Daemonize thread
-        thread.start()
-        LOGGER.info("Step 1: Successfully started WRITEs-READs-verify in background")
+        LOGGER.info("Waiting for %s seconds to perform some WRITEs",
+                    HA_CFG["common_params"]["30sec_delay"])
+        time.sleep(HA_CFG["common_params"]["30sec_delay"])
+        LOGGER.info("Step 5: Perform READs and verify DI on the written data in background")
+        output_rd = Queue()
+        args = {'s3userinfo': list(users.values())[0], 'log_prefix': test_prefix_read,
+                'nclients': 1, 'nsamples': 7, 'skipwrite': True, 'skipcleanup': True,
```

Comment:
> nclients for write for this read seems to have set to 7 and now 1? Also can we keep lower value like 2/2 as we had decided before as with K pods going down, it will create perf drop.

This thread will run in background. WRITEs for this READs we wanted to make faster but here, we have to keep IOs running till K pods shutdown hence lower value ie.1

Yes. performance will drop but we can take a look during execution and then decide with the count as we have to make sure that IOs should be running to meet our test condition, where IOs should be parallel while K pods goes down